pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -798,7 +798,8 @@ func (c *Reconciler) createTaskRuns(ctx context.Context, rpt *resources.Resolved
 	ctx, span := c.tracerProvider.Tracer(TracerName).Start(ctx, "createTaskRuns")
 	defer span.End()
 	var taskRuns []*v1beta1.TaskRun
-	matrixCombinations := rpt.PipelineTask.Matrix.FanOut()
+	rpt.PipelineTask.Matrix.FanOut()
```

Comment:
it seems you meant to delete this line, the function is actually used in 802 below
```suggestion

```